detox/src/utils/exec.js

```diff
@@ -33,12 +32,12 @@ async function execWithRetriesAndLogs(bin, options = {}) {
       } else if (statusLogs.retrying) {
         _logRetrying(log, cmd, retryNumber, lastError);
       }
-      result = await traceCall(`cmd: ${cmd}`, () => exec(cmd, { timeout }));
+      result = await exec(cmd, { timeout });
```

Comment:
@noomorph I had to remove `traceCall()` because this is now called from the global Jest context - where trace is not initialized (and thus crashes), and doesn't get collected.
We should think of a way to fix this. Possibly, we could initialize it in some fake manner from the global context, just so as to avoid crashing.